### DISCUSSION TITLE
✅ 🚀 Feature Request: Dynamic content for nested accordion

### DISCUSSION THREAD

**Discussion by creativeSpecies** [elementor/elementor#23889] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

In a same website, I can have a nested accordion, which works great for standard pages. But sometimes, I also need a dynamic one, especially in posts templates, where each post it's applied, can have a different number of items.

### Describe the solution you'd like

It would be nice if there were a conditional rule like 'dynamic source' and we can have a choice like shortcodes, acf fields or similar, etc

### Describe alternatives you've considered

I can solve it with a custom widget or hooks

### Additional context

Already tried to extend the Nested_Accordion class, but the protected function render() doesn't seem to be called at all.

**Comment by nicholaszein** [comment]
Hello @creativeSpecies!

Thank you for your suggestion! 🙏 

This is already possible to achieve using The Loop Grid widget with a Nested Accordion inside. All you have to do is to set a Dynamic Tag for a single nested accordion item title, and then design your dynamic content using any Elementor widget that supports Dynamic Tags. Then all you have to do is to set a Query for your Loop Grid template and the accordion will appear as expected. That's the beauty of Nested Elements. They already allow you to add any content inside them. You don't really need a setting in the widget itself to achieve what you want.

I'm closing this request as solved.

Kind regards
